src/miral/wayland_extensions.cpp

```diff
@@ -98,6 +96,16 @@ miral::WaylandExtensions::WaylandExtensions(std::string const& default_value) :
 }
 
 auto miral::WaylandExtensions::supported_extensions() const -> std::string
+{
+    return self->available_extensions.substr(0, self->available_extensions.size()-1);
```

Comment:
It appears this relies on `self->available_extensions` ending with a trailing colon. As this doesn't seem to be enforced (as far as I can tell), we should probably check before we chop it off.

Alternatively, we could *not* use colon separated strings internally, and not have this problem.